consensus/integration_test_types.go

```diff
@@ -167,6 +167,9 @@ func (node *validatorNode) ProcessBlock(incomingBlock *block) {
 				return BlockWithValidatorList{bb, node.getValidators()}
 			},
 		),
+		// TODO: replace with values from snapshot global params
```

Comment:
No need for this TODO. This file is just used for unit / integration testing within the consensus module itself.